store/alert_def_store.go

```diff
@@ -0,0 +1,8 @@
+package store
+
+import "Price_Notification_System/models"
+
+type AlertDefStore interface {
+	AddAlert(item string, newAlertDef models.AlertValues)
+	ProcessAlerts(alertToProcess models.AlertDef) (data []models.HistoricalTradeAlertReturned, err error)
```

Comment:
processing / business logic is not a part of a store. 

storage is typically concerned about memory / storing things in persistence. 
the service layer implements the logic.